manage.py

```diff
@@ -33,5 +34,7 @@ def make_shell_context() -> dict:
 manager.add_command('runserver', Server())
 manager.add_command('shell', Shell(make_context=make_shell_context))
 
+[manager.add_command('delete', c) for c in app.commands]
```

Comment:
Doesn't the command need to define what its name is? (Aren't all commands going to be set as `delete`?)